Sami/RemoteRepository/GitHubRemoteRepository.php

```diff
@@ -15,7 +15,7 @@ class GitHubRemoteRepository extends AbstractRemoteRepository
 {
     public function getFileUrl($projectVersion, $relativePath, $line)
     {
-        $url = 'https://github.com/'.$this->name.'/blob/'.$projectVersion.$relativePath;
+        $url = 'https://github.com/'.$this->name.'/blob/'.str_replace('\\', '/',$projectVersion.$relativePath);
```

Comment:
Ah, then to be consistent use solution from there. But missing space change still remains relevant.